shared/src/main/kotlin/com/egm/stellio/shared/model/ApiExceptions.kt

```diff
@@ -2,24 +2,129 @@ package com.egm.stellio.shared.model
 
 import com.apicatalog.jsonld.JsonLdError
 import com.apicatalog.jsonld.JsonLdErrorCode
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ProblemDetail
+import org.springframework.http.ResponseEntity
+import java.net.URI
+
+const val DEFAULT_DETAIL = "If you have difficulty identifying the issue," +
+    "you can check common problem on https://stellio.readthedocs.io/en/latest/TROUBLESHOOT.html" +
+    "If you believe this is a bug, you can report it by creating an issue on" +
+    "https://github.com/stellio-hub/stellio-context-broker"
```

Comment:
```suggestion
val DEFAULT_DETAIL =
    """
        If you have difficulty identifying the exact cause of the error,
        please check the list of some usual causes on https://stellio.readthedocs.io/en/latest/TROUBLESHOOT.html.
        If the error is still not clear or if you think it is a bug, feel free to open an issue on
        https://github.com/stellio-hub/stellio-context-broker
    """.trimIndent()
```

Reply:
Sadly the responseEntity mapper dislike line breaks. So this put "\n" in the return message.
```json
    "detail": "If you have difficulty identifying the exact cause of the error,\nplease check the list of some usual causes on https://stellio.readthedocs.io/en/latest/TROUBLESHOOT.html.\nIf the error is still not clear or if you think it is a bug, feel free to open an issue on\nhttps://github.com/stellio-hub/stellio-context-broker",
```
maybe it can be configured to work properly.

Reply:
don't bother with this, go back to the string concat version (but keep with the suggested content :) )

Reply:
We can't remove them because the encoding of the line break come from the json specification : https://www.[ietf.org/rfc/rfc4627.txt](https://www.ietf.org/rfc/rfc4627.txt)